web/components/editor/contract-mention-suggestion.ts

```diff
@@ -15,12 +13,11 @@ export const contractMentionSuggestion: Suggestion = {
   // Note (James): After recreating yarn.lock, this line had a type error, so I cast it to any...
   pluginKey: new PluginKey('contract-mention') as any,
   items: async ({ query }) =>
-    orderBy(
-      (await getCachedContracts()).filter((c) =>
-        searchInAny(query, c.question)
-      ),
-      [(c) => [c.question].some((s) => beginsWith(s, query))],
-      ['desc']
-    ).slice(0, 5),
+    (
+      await trendingIndex.search(query, {
+        hitsPerPage: 5,
+        removeStopWords: true,
+      })
+    ).hits,
```

Comment:
Cool that was easier than one might think. Should have done this earlier haha.

Reply:
Seems like the search index `searchClient.initIndex(searchIndexName)` would be better than the trending index?